src/commands/build.js

```diff
@@ -59,7 +60,7 @@ exports.handler = async ({
 
   const mode = production ? 'production' : 'development';
   // Use env variables if working on Webpack >=5.
-  const projectsList = projects.split(',').filter((item) => item.length > 0);
+  const projectsList = projects.split(',').filter((item) => item.length > 0).map((item) => item.split('@')[0]);
   const hasTargetDirs = dirsExist(targetDirs);
   const isAllProjects = (site || hasTargetDirs) && !projects;
```

Comment:
I found that `@entrypoint` can be treat by this logic as a project. So with an entrypoint only set in the comment, this will return a false-positive. Seems like we can get around this by adding another logical operator against `projectsList` length.

```suggestion
  const isAllProjects = (site || hasTargetDirs) && (!projects || projectsList.length == 0);
```

There may be a cleaner, less convoluted way of doing this by my quick test allowed me to build entrypoints only.

Reply:
Made this change in my latest, commit. Wanted to test locally.